spec/qonto/client_spec.rb

```diff
@@ -102,6 +103,23 @@
       response = client.list_transactions(bank_account: account)
 
       expect(response.first).to be_a(Qonto::Model::Transaction)
+      expect(response.first.instance_variables).to eq(%i[@amount
```

Comment:
I think that what we really want to test here, is that each attribute present in the JSON response is accessible through the model interface, and that it has been set at the good value.

What do you think of instead adding a test per attribute we want to test ? This is a but longer to write, but may be safer on a long term point of view (= we are sure that everything is present and correctly set). And maybe easier to read.

Example:
```
expect(response.first.amount).to eq(0.42)
expect(response.first.amount_cents).to eq(42)
# ...
```

Do you agree ?